### ISSUE TITLE
How to reproduce the results on the HO-3D dataset

### ISSUE THREAD

**Issue by hxwork** [namepllet/HandOccNet#12]
Hi,

Thanks for making your code publicly available, and this work is really impressive! I have tried to load the pre-trained model and found that its performance on the HO-3D dataset is slightly worse than that reported in the paper. Besides, I am trying to reproduce the results on the HO-3D dataset by re-training the model using this code. However, I have two problems:
1. The training epoch in the `config.py` is 70, but the pre-trained model is `pred80.pth.tar`. Does that mean I need to train more epochs than described in the `config.py`?
2. I also tried to train a new model using the code in this repo without changing anything, but the results of both models trained with 70 and 80 epochs are not consistent with that shown in the paper, as shown in the table below.
![image](https://user-images.githubusercontent.com/46584121/189476416-6cf35870-fbc4-412a-afea-1598e9439b83.png)
So I'd like to ask how to reproduce the results.

Best regards,
Hao

**Comment by Strive9881** [comment]
Hello, How long did it take you to train for 80 epochs and what gpu did you use?

**Comment by hxwork** [comment]
Hi,

It takes ~10 hours to train for 70 epochs and another 1.5h to train for the remaining 10 epochs, and I used 4 Titan V GPUs to train the network.

**Comment by namepllet** [comment]
We trained our model using 4 2080Ti and it takes ~10 hours.

In our experiments, mean joint error are same at 70 and 80 epochs and other metrics are slightly better at 80 epochs.

**Comment by Strive9881** [comment]
Hi，
I have the same problem with the reproduction accuracy, have you solved it?

**Comment by hxwork** [comment]
Hi,

Unfortunately, I have no idea to find where the problem is since I have not changed a line of the code (except the directory path of the dataset) and the results are just shown above. Maybe you can compare your results with mine and see if there is any difference between them.

Best,
Hao

**Comment by Strive9881** [comment]
Hi,
This is the result of my reproduction.
![image](https://user-images.githubusercontent.com/51118126/192556957-cb8e6113-d2e9-4260-96bd-f13af45f4b2b.png)

**Comment by hxwork** [comment]
Got it. Your results are slightly better than mine, but it looks like there is still a large margin between the results of the re-trained model and those shown in the final version paper. Hope you can find out where the problem is. : )

**Comment by Strive9881** [comment]
Ok, I will keep you informed if I resolve it.
There is another question I'd like to ask.
HandOccNet has documented experimental results of the Pose2Mesh model on the HO3D dataset, but I found that the Pose2Mesh paper did not experiment on the HO3D dataset. Do you know the source of this experimental results?

**Comment by hxwork** [comment]
Thanks. For the question you mentioned above, I am sorry that I don't know. And I notice that you have opened another issue about this. Maybe you should wait for the author's response.

**Comment by JT-Sun** [comment]
Hi,
I trained the model for 70 epoh on the HO3D dataset, but the result sames a far cry from your results. Do anyone know the reason?
F_aligned@5.0mm = 0.345
F_aligned@15.0mm = 0.845
best!

**Comment by kyxxc** [comment]
Hello. Regarding the final F_aligned@5.0mm , F_aligned@15.0mm How is it viewed

**Comment by SUOXIAOXI** [comment]
Hello, does the HO3D dataset have to be evaluated on the platform when testing, the limited file size on the platform is 300M, but the JSON I got has 500M, what should I do in this situation？

**Comment by M-leng** [comment]
Hi, My result is also bad. I trained Handoccnet on Ho-3D for 70 epochs, on one GPU. My result is similar to it.

How to train the model to get the result of the paper?

![image](https://github.com/user-attachments/assets/bdeab6d0-cec0-4437-824e-3fab4e7adf5e)


> Hi, I trained the model for 70 epoh on the HO3D dataset, but the result sames a far cry from your results. Do anyone know the reason? [F_aligned@5.0mm](mailto:F_aligned@5.0mm) = 0.345 [F_aligned@15.0mm](mailto:F_aligned@15.0mm) = 0.845 best!
